setup.cfg

```diff
@@ -96,10 +96,17 @@ nibabel =
 max-line-length = 100
 ignore = D100,D101,D102,D103,D104,D105,D200,D201,D202,D204,D205,D208,D209,D210,D300,D301,D400,D401,D403,E24,E121,E123,E126,E226,E266,E402,E704,E731,F821,I100,I101,I201,N802,N803,N804,N806,W503,W504,W605
 exclude =
+    venv
+    build
     *test*
     *sphinx*
     nibabel/externals/*
     */__init__.py
+    # temporary -- should be removed in another PR
```

Comment:
Need a follow-up PR to clean up the remaining issues. I added these as ignores so that local flake8 passes. These were being covered up by the configuration of pep8speaks [only looking at the diff](https://github.com/nipy/nibabel/blob/master/.pep8speaks.yml#L2)